src/plugins/profiling/common/index.ts

```diff
@@ -88,3 +88,18 @@ export function timeRangeFromRequest(request: any): [number, number] {
   const timeTo = parseInt(request.query.timeTo!, 10);
   return [timeFrom, timeTo];
 }
+
+export const now = (label?: string) => {
+  if (label) {
+    console.log('Started', label);
+  }
+  return new Date().getTime();
+};
+
+export const elapsed = (start: number, label?: string) => {
+  const duration = new Date().getTime() - start;
+  if (label) {
+    console.log(label, `${duration / 1000}s`);
+  }
+  return duration;
+};
```

Comment:
This is only a comment and not a suggestion for this PR. But if we decide to keep the performance timings around long term, I wanted to mention the Performance API in NodeJS [1]. It uses the same Performance API as implemented in modern browsers [2].

Aside from using standardized APIs, one possible benefit is that multiple measurements can be batched together, allowing us to log measurements in one message or possibly package up the measurements in the API response.

We could also eventually use `Logger` in `./src/core/server`.

1. https://nodejs.org/docs/latest-v16.x/api/perf_hooks.html
2. https://developer.mozilla.org/en-US/docs/Web/API/Performance

Reply:
Thanks for the hints !
Currently (just for quick testing), the Performance API is a bit too heavy.
Later on we IMO should use the Logger (already had a chat with @inge4pres about it).

Reply:
Yep, my preference would be to pass around the Logger that comes with the plugin, it seems the most natural way